### DISCUSSION TITLE
How can I enable the full article in rss feed instead of excerpt?

### DISCUSSION THREAD

**Discussion by Natenom** [razonyang/hugo-theme-bootstrap#401] [Q&A] [answered]
I would like to have the full articles in the feed. Not just an excerpt.

How can I achieve this?

**Comment by razonyang** [comment, marked as answer]
This theme use the default [RSS template](https://gohugo.io/templates/rss/#the-embedded-rssxml). You'll need to create your own RSS template(`layouts/_default/rss.xml`). And modify the `<description>{{ .Summary | html }}</description>` to `<description>{{ .Content| html }}</description>`.

**Reply by Natenom** [reply]
Thank you :) Now I know where to look for files that are not included in a Theme.
